cms/static/js/views/settings/main.js

```diff
@@ -139,14 +139,16 @@ var DetailsView = ValidatingView.extend({
         var now = new Date(),
             hours = now.getUTCHours(),
             minutes = now.getUTCMinutes(),
-            currentTimeText = gettext('%(hours)s:%(minutes)s (current UTC time)');
+            currentTimeText = StringUtils.interpolate(
```

Comment:
`./scripts/safe_template_linter` was invalidated because the use of interpolate and it suggested 
```
javascript-interpolate: Use StringUtils.interpolate() or HtmlUtils.interpolateHtml() as appropriate.
```

I have updated the main.js. kindly have a look. 

@robrap @mushtaqak